src/Char.cpp

```diff
@@ -559,3 +559,9 @@ void Char::try_give_item_to(Object *object, Char *victim) {
     handle_corpse_summoner(this, victim, object);
     mprog_give_trigger(victim, this, object);
 }
+
+std::string_view Char::describe(const Char &to_describe) const noexcept {
```

Comment:
much nicer!

Reply:
Can be the basis of a lot of things that can be described too, like ...describe(Room &room) etc.

Also I wondered about trying to find a way to "just" "know" which Char is active in e.g. `send_to()` etc, so that ideally we could just do `send_line("{} is here.", other);` and have the formatter "know" that it's formatting for "*this" etc. but I couldn't see how to do that without an evil global.